src/components/careers/Benefits/index.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react'
+import styled from 'styled-components'
+import BenefitsCard from './Card'
+import ContentWrapper from '../../Layout/ContentWrapper'
+
+import BookIcon from '../../../assets/icon-book.svg'
+import BulbIcon from '../../../assets/icon-bulb.svg'
+import BuildingIcon from '../../../assets/icon-building.svg'
+import CalendarIcon from '../../../assets/icon-calendar.svg'
+import TeamIcon from '../../../assets/icon-team.svg'
+import LaptopIcon from '../../../assets/icon-laptop.svg'
+import TokensIcon from '../../../assets/icon-tokens.svg'
+import GlobeIcon from '../../../assets/icon-globe.svg'
+import BgGray from '../../../assets/bg-gray.svg'
+import { SectionTitle, Text } from '../styled'
+import { sizes } from '../../../styles/styleUtils'
+
+const Grid = styled.div`
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 24px;
+
+  @media screen and (max-width: ${sizes.md}px) {
+    grid-template-columns: repeat(2, minmax(0, 1fr));
+    gap: 20px;
+  }
+`
+
+const SectionHeader = styled.div`
+  margin: 0 auto 80px;
+  max-width: 732px;
+  text-align: center;
+`
+
+const SBgGray = styled(BgGray)`
+  position: absolute;
+  top: 50px;
+  width: 2000px;
+  transform: translateX(-50%);
+  left: 50%;
+  min-width: 100%;
+
+  @media screen and (max-width: 980px) {
```

Comment:
We should probably extract this to a constant as well.

Reply:
We have these breakpoint values
https://github.com/gnosis/safe-landing-page/blob/a9884a60507253ce0382cd8d2256f35588ac9960/src/styles/styleUtils.ts#L4-L9

could I update the `lg` value to _980_ and use that value? Wdyt @germartinez